test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/MyBasePageOfTModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Mvc.Razor;
+
+namespace RazorCodeGen
+{
+    public class MyBasePage<TModel> : RazorPage<TModel>
```

Comment:
The tests don't need to be runnable/compile. I'd nuke this class for the sake of simplicity.